cmd/git_utils_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd_test
+
+import (
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+
+	cmd "github.com/appsody/appsody/cmd"
+	"github.com/appsody/appsody/cmd/cmdtest"
+)
+
+func TestStringBefore(t *testing.T) {
+	var StringBeforeTests = []struct {
+		testName       string
+		value          string
+		searchString   string
+		expectedOutput string
+	}{
+		{"Empty values", "", "", ""},
+		{"Non-empty values", "teststring", "string", "test"},
```

Comment:
one more test where the search string is not in the string value, should return empty string

Reply:
Should it return an empty string? Because when I just tested it, it returns the `value` string in full. 

For example, in this case
```
output := cmd.StringBefore("This is what I have", "But this is what I am looking for")
```
`output` = "This is what I have"